dev/make-distribution.sh

```diff
@@ -29,7 +29,7 @@ set -e
 set -x
 
 # Figure out where the Spark framework is installed
-SPARK_HOME="$(cd "`dirname "$0"`"; pwd)"
+SPARK_HOME="$(cd "`dirname "$0"`/.."; pwd)"
```

Comment:
Oh never mind, spark-ec2 [actually doesn't use make-distribution.sh](https://github.com/amplab/spark-ec2/blob/998358daebce25ff49907779a3fa391289ac4c80/spark/init.sh#L10-L23), so this shouldn't be a problem.